test/e2e/autotls/auto_tls_test.go

```diff
@@ -154,7 +183,20 @@ func routeTrafficHTTPS(route *servingv1.Route) (bool, error) {
 			return false, nil
 		}
 	}
-	return route.Status.IsReady() && true, nil
+	return route.Status.IsReady(), nil
+}
+
+func routeUrlHTTP(route *servingv1.Route) (bool, error) {
+	return route.Status.URL.URL().Scheme == "http", nil
+}
+
+func routeTLSDisabled(route *servingv1.Route) (bool, error) {
+	for _, cond := range route.Status.Conditions {
```

Comment:
r.Status.GetCondition here.

Reply:
ah of course there's a method for this :) thanks!